ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/ExceptionScopeField.tsx

```diff
@@ -1,23 +1,35 @@
 import React from 'react';
 import { FormGroup, FormGroupProps, Radio } from '@patternfly/react-core';
 
-import { ScopeContext } from './utils';
+import { useFormik } from 'formik';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Is disable comment a left over from skeleton without form state?

Reply:
It is not, I believe this is a bug in @typescript-eslint. The type is used [below](https://github.com/stackrox/stackrox/pull/8161/files/26964fe7630cff8dfab3f6d38341a58ccca9f3b4#diff-b0209778d5b97303aec0214871c9032816187e26954201eeccdff5d4797e55f4R14), but removing this comment causes the linter to fail with an error.